README.md

```diff
@@ -74,17 +82,35 @@ Nullable bool.
 Will marshal to false if null. `false` produces a null Float. Null values and zero values are considered equivalent.
 
 #### zero.Time
+Nullable time.
 
 Will marshal to the zero time if null. Uses `time.Time`'s marshaler.
 
-### Can you add support for other types?
+#### zero.Value[`T`]
+Generic nullable value.
+
+Will marshal to zero value if null. `T` is required to be a comparable type. Does not implement `encoding.TextMarshaler`.
+
+## About
+
+### Q&A
+
+#### Can you add support for other types?
 This package is intentionally limited in scope. It will only support the types that [`driver.Value`](https://godoc.org/database/sql/driver#Value) supports. Feel free to fork this and add more types if you want.
 
 ### Can you add a feature that ____?
 This package isn't intended to be a catch-all data-wrangling package. It is essentially finished. If you have an idea for a new feature, feel free to open an issue to talk about it or fork this package, but don't expect this to do everything.
 
 ### Package history
-*As of v4*, unmarshaling from JSON `sql.NullXXX` JSON objects (ex. `{"Int64": 123, "Valid": true}`) is no longer supported. It's unlikely many people used this, but if you need it, use v3.
+
+#### v5
+- Now a Go module under the path `github.com/guregu/null/v5`
+- Added missing types from `database/sql`: `Int32, Int16, Byte`
+- Added generic `Value[T]` embedding `sql.Null[T]`
+
+#### v4
+- Available at `gopkg.in/guregu/null.v4`
+- Unmarshaling from JSON `sql.NullXXX` JSON objects (e.g. `{"Int64": 123, "Valid": true}`) is no longer supported. It's unlikely many people used this, but if you need it, use v3.
```

Comment:
This section can be removed (or updated to explain this is a simple fork of `github.com/guregu/null` that contains additional types.